jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/importer/ImporterDirectoryWatcher.java

```diff
@@ -29,7 +29,7 @@
 public class ImporterDirectoryWatcher implements Runnable {
    
    public enum ImporterFileType {
-      LEAR_LOG_FILE, JSON_FILE
```

Comment:
What is the diff between JSON and LOG file?

Reply:
The main difference is that JSON is uncompressed and the LOG file is a gzip binary file. I think the old DSRC Lear devices used to have this as a configurable setting. Current devices only support the gzip binary file output.